build.rs

```diff
@@ -40,7 +40,7 @@ fn main() -> std::io::Result<()> {
                             .to_str()
                             .unwrap()
                             .to_string()
-                            .replace('/', "::")
+                            .replace(std::path::MAIN_SEPARATOR, "::")
```

Comment:
Good one!! Guess this would make it work on Windows as well.

Reply:
Yes sir. One of our teammates uses Windows and it works fine for them now.